pkg/deploy/identity-provider/deployment_keycloak.go

```diff
@@ -489,13 +490,18 @@ func getSpecKeycloakDeployment(
 		" && /opt/jboss/docker-entrypoint.sh -b 0.0.0.0 -c standalone.xml"
 	command += " -Dkeycloak.profile.feature.token_exchange=enabled -Dkeycloak.profile.feature.admin_fine_grained_authz=enabled"
 	if cheFlavor == "codeready" {
+		addUsernameReadonlyTheme := "if [ ! -d /opt/eap/themes/codeready-username-readonly ]; then" +
```

Comment:
I love that this project allows 'if product then...' branches -- makes the sync/fork/massaging easier downstream :)